corehq/apps/reports_core/static/reports_core/js/base_template_new.js

```diff
@@ -50,8 +50,9 @@ hqDefine('reports_core/js/base_template_new', function () {
         };
 
         var successCallback = function (data) {
-            if (data.error) {
-                $('#error-message').html(data.error);
+            if (data.error || data.error_message) {
```

Comment:
nit: we still checking for error?

Reply:
I don't think it's technically necessary, but because there's many places this code is being used from, I thought I'd rather be safe and still check for it. I don't want to break other code by accident.

Reply:
Sounds good to me